.github/workflows/tests.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       options: --user 1001
     services:
       meilisearch:
-        image: getmeili/meilisearch:v1.0.2
+        image: getmeili/meilisearch:v1.5
```

Comment:
@brunoocasali @mdubus shouldn't we use `latest` instead like in all the integration? We could avoid this kind of manual update

Reply:
Also the risk currently we had/have here is the test are green for v1.0.2 but not for v1.6.0 when out and we wouldn't even notice it.

Reply:
Thanks for your insight @curquiza 🙏 
What we usually did before was update the image with the pre-release Meilisearch version, that way we were almost sure that there won't be no problem. 
I'm pretty sure that's why we never put `latest` in the CI for the meilisearch version. 
Would that work for you if I update the `v1.6.0-rc.0` tag?

Reply:
Gotcha 👍  Thanks for taking the time to explain this to me. 
I updated the CI file to use the `latest` instead :)